packages/components/src/components/chip/chip.tsx

```diff
@@ -35,8 +35,8 @@ export class Chip {
   @Prop() selected?: boolean = false;
   /** (optional) chip aria-role */
   @Prop() ariaRoleTitle?: string;
```

Comment:
Instead of the ternary below when defining the `role`,  why not add a default value to the prop? would that work?

```ts
@Prop() ariaRoleTitle?: 'switch' | 'radio' | ... = 'switch';
```

and I would use these as possible types, [according to mdn](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-checked#associated_roles):
- checkbox
- menuitemcheckbox
- menuitemradio
- option
- radio
- switch

---
btw it's a shame this is called `ariaRoleTitle` instead of `innerAriaRole`, no? 😕

Reply:
> Instead of the ternary below when defining the role, why not add a default value to the prop? would that work?

this is fixed

> btw it's a shame this is called ariaRoleTitle instead of innerAriaRole, no? 😕

yes but i guess changing this would be a breaking change if people are using the old prop name...